environment/prod/deployment/beta/docker-compose.beta.yml

```diff
@@ -14,7 +14,7 @@ volumes:
 
 services:
   keating-beta-app:
-    image: ghcr.io/blumilksoftware/keating:beta
+    image: registry.blumilk.pl/internal-public/keating:beta
```

Comment:
This should be added to the Renovate config as ignore deps.

Reply:
Added